packages/terra-time-input/src/terra-dev-site/test/time-input/twelve-hour/twelve-hour.test.common.scss

```diff
@@ -0,0 +1,5 @@
+:local {
```

Comment:
we can re-use the same class here also : [time-input.test.common.scss](https://github.com/cerner/terra-framework/blob/2fcbf0eb8727d4d15b8c0da41131181555c53c06/packages/terra-time-input/src/terra-dev-site/test/time-input/time-input/time-input.test.common.scss#L2)

Reply:
fixed here https://github.com/cerner/terra-framework/pull/692/commits/6c706bab37454c6d9c4fb8a8ee2678b5ccc57890.